charts/r2devops/templates/configmap.yaml

```
@@ -2,29 +2,43 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
+  annotations:
+    {{- range $key, $val := .Values.additionalAnnotations }}
+    {{ $key }}: {{ $val | quote }}
+    {{- end }}
   labels:
     app: {{ .Values.front.name }}
     app.kubernetes.io/name: {{ .Values.front.name }}
+    {{- range $key, $val := .Values.additionalLabels }}
+    {{ $key }}: {{ $val | quote }}
+    {{- end }}
   name: {{ .Values.front.name }}
 data:
   config.body: |
     {
-      "appTitle": "R2Devops",
+      "appTitle": "R2Devops BAS",
       "apiUrl": "https://{{ .Values.jobs.host }}/api",
       "gitLabApiUrl": "{{ .Values.gitlab.domain }}",
       "selfHosted": true,
       "docUrl": "https://docs.r2devops.io",
-      "debug": false,
+      "debug": true,
       "allowExternalQueries": true
     }
 
 ---
 apiVersion: v1
 kind: ConfigMap
 metadata:
+  annotations:
+    {{- range $key, $val := .Values.additionalAnnotations }}
+    {{ $key }}: {{ $val | quote }}
+    {{- end }}
+  labels:
+    {{- range $key, $val := .Values.additionalLabels }}
+    {{ $key }}: {{ $val | quote }}
+    {{- end }}
```

Avoid empty keys if no `additionalAnnotations` or `additionalLabels`:

```suggestion
  {{- with .Values.additionalAnnotations }}
  annotations:
    {{- range $key, $val := . }}
    {{ $key }}: {{ $val | quote }}
    {{- end }}
  {{- end }}
  {{- with .Values.additionalLabels }}
  labels:
    {{- range $key, $val := . }}
    {{ $key }}: {{ $val | quote }}
    {{- end }}
  {{- end }}
```